### PR TITLE
Feature/excel parse

### PR DESCRIPTION
**Pull request by netashefer**

### CODE REVIEW

File: src/helpers/xls-reader.ts [hunk 1]
```diff
@@ -20,3 +21,25 @@ export const convertToJson = (csv: string): Table => {
 
     return { data: result, schema: headers };
 }
+
+export const parseTable = (table: Table) => { // should be on processing-service
+    const columnParsingMap: Record<string, (value: string) => string | Date | number> = {};
```
Comment (YahelR) [inline src/helpers/xls-reader.ts]: Can you create types for these?

File: src/helpers/xls-reader.ts [hunk 2]
```diff
@@ -20,3 +21,25 @@ export const convertToJson = (csv: string): Table => {
...
+    table.schema.forEach(column => {
+        if (column.includes("(DATE)")) {
```
Comment (YahelR) [inline src/helpers/xls-reader.ts]: move to consts

File: src/helpers/xls-reader.ts [hunk 3]
```diff
@@ -20,3 +21,25 @@ export const convertToJson = (csv: string): Table => {
...
+            columnParsingMap[column] = (d: string) => new Date(d)
```
Comment (YahelR) [inline src/helpers/xls-reader.ts]: rename d to date

**Review by YahelR — COMMENTED** [review]
I don't mind if you won't change these

File: src/components/ManagerPanel/ManagerPanel.tsx [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
-import ReactTooltip from 'react-tooltip';
-import classNames from 'classnames';
-import Tooltip from '@mui/material/Tooltip';
 
-import './ManagerPanel.scss';
 import DataManager from '../DataManager/DataManager';
+import './ManagerPanel.scss';
 
 interface ManagerPanelProps {
     addDataInstanceTable: (table: Table, info: any) => void;
```
Comment (YahelR) [inline src/components/ManagerPanel/ManagerPanel.tsx:13]: also what is info supposed to be, is it really any?

Comment (netashefer) [reply src/components/ManagerPanel/ManagerPanel.tsx:13]: its like Blob, I believe it will be changed when we insert recoil

**Pull request merged**